modules/switcher.js

```diff
@@ -0,0 +1,29 @@
+// display current date
+/* eslint-disable no-undef */
```

Comment:
[OPTIONAL] Rather than disable the Linters, strive to fix them as this makes you more accustomed to the style guide of the tech.